shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -883,7 +874,7 @@ - (UInt32)adjustModifiers:(nonnull FlutterUIPressProxy*)press API_AVAILABLE(ios(
 
 namespace {
 void HandleResponse(bool handled, void* user_data) {
-  FlutterKeyPendingResponse* pending = reinterpret_cast<FlutterKeyPendingResponse*>(user_data);
+  FlutterKeyPendingResponse* pending = (__bridge_transfer FlutterKeyPendingResponse*)user_data;
```

Comment:
Same, tests crashed with just a `__bridge`.

Reply:
What crash did you get? Does the owner of user_data release it too?

Reply:
Oh i saw the caller uses `autorelease` to keep it alive for current runloop. So in ARC it makes sense to bridge retain, and bridge release afterwards.

Reply:
The crash was an overrelease, it was zombied.

Reply:
ya what you did here should be fine. The caller site doesn't release it, so won't be over-released.